src/main/kotlin/com/grepp/nbe1_3_team04/vote/repository/CustomVoteRepositoryImpl.kt

```diff
@@ -0,0 +1,61 @@
+package com.grepp.nbe1_3_team04.vote.repository
+
+import com.grepp.nbe1_3_team04.global.domain.IsDeleted
+import com.grepp.nbe1_3_team04.vote.domain.*
+import com.querydsl.jpa.impl.JPAQueryFactory
+import java.util.*
+
+class CustomVoteRepositoryImpl(private val queryFactory: JPAQueryFactory) : CustomVoteRepository {
+    override fun findNotDeletedVoteById(id: Long): Vote? {
+        return queryFactory.select(QVote.vote)
+                .from(QVote.vote)
+                .where(
+                    QVote.vote.voteId.eq(id)
+                        .and(QVote.vote.isDeleted.eq(IsDeleted.FALSE))
+                )
+                .fetchOne()
+
+    }
+
+    override fun findOpenedVotes(): List<Vote> {
+        return queryFactory.select(QVote.vote)
+            .from(QVote.vote)
+            .where(
+                QVote.vote.isDeleted.eq(IsDeleted.FALSE)
+                    .and(QVote.vote.voteStatus.eq(VoteStatus.OPENED))
+            )
+            .fetch()
+    }
+
+    override fun choiceMemberCountByVoteId(voteId: Long): Long? {
+        return queryFactory.select(QChoice.choice.memberId.countDistinct())
+            .from(QVote.vote).join(QVoteItem.voteItem).on(QVoteItem.voteItem.vote.eq(QVote.vote))
+            .leftJoin(QChoice.choice).on(QChoice.choice.voteItemId.eq(QVoteItem.voteItem.voteItemId))
+            .where(
+                QVote.vote.voteId.eq(voteId)
+                    .and(QVote.vote.isDeleted.eq(IsDeleted.FALSE))
+            )
+            .fetchOne()
+    }
+
+    override fun findRecentlyVoteByTeamId(teamId: Long): Vote? {
+        return queryFactory.select(QVote.vote)
+            .where(
+                QVote.vote.isDeleted.eq(IsDeleted.FALSE)
+                    .and(QVote.vote.voteStatus.eq(VoteStatus.CLOSED))
+            )
+            .from(QVote.vote)
+            .orderBy(QVote.vote.updatedAt.desc())
+            .fetchFirst()
```

Comment:
```suggestion
        return queryFactory.select(QVote.vote)
            .from(QVote.vote)
            .where(
                QVote.vote.isDeleted.eq(IsDeleted.FALSE)
                    .and(QVote.vote.voteStatus.eq(VoteStatus.CLOSED))
            )
            .orderBy(QVote.vote.updatedAt.desc())
            .fetchFirst()
```
그냥 특별한 내용은 아니고, 순서가 눈에 띄었을 뿐 입니다 ㅎㅎ...

Reply:
오잉... 그렇네요 뭐지 어떻게 from이랑 where절이 바뀌었을까요 원래 이랬나.. 감사합니다 당황스럽네요 ㅋㅋ